<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripleDeleteCondDlg</class>
 <widget class="QDialog" name="TripleDeleteCondDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Delete triples according to</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond1"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT1">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond2"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT2">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 4</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond4"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT4">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>AND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 3</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond3"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT3">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TripleDeleteCondDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TripleDeleteCondDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond1</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TripleDeleteCondDlg</receiver>
   <slot>changeIdx_1(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TripleDeleteCondDlg</receiver>
   <slot>changeIdx_2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>-29</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond3</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TripleDeleteCondDlg</receiver>
   <slot>changeIdx_3(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond4</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TripleDeleteCondDlg</receiver>
   <slot>changeIdx_4(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>-10</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeIdx_1(int)</slot>
  <slot>changeIdx_2(int)</slot>
  <slot>changeIdx_3(int)</slot>
  <slot>changeIdx_4(int)</slot>
 </slots>
</ui>
